motr/client.c

```diff
@@ -428,7 +428,7 @@ void m0_obj_init(struct m0_obj *obj,
 
 #ifdef OSYNC
 	m0_mutex_init(&obj->ob_pending_tx_lock);
-	ospti_tlist_init(&obj->ob_pending_tx);
+	spti_tlist_init(&obj->ob_pending_tx);
```

Comment:
This was a typo bug.
Why is it not found before? Because this condition is never satisfied?

Reply:
I suppose so, yes. Thanks @huanghua78 for review!